proto/tendermint/types/types.proto

```diff
@@ -201,3 +210,37 @@ message TxProof {
   bytes                   data      = 2;
   tendermint.crypto.Proof proof     = 3;
 }
+
+
+message Evidence {
+  oneof sum {
+    DuplicateVoteEvidence     duplicate_vote_evidence      = 1;
+    LightClientAttackEvidence light_client_attack_evidence = 2;
+  }
+}
```

Comment:
also wondering why we moved evidence here from evidence.proto? is this better practice for proto files

Reply:
We had to do this b/c Proposal needs Evidence.proto, but that would have resulted in a circular dependency:
`types.proto (proposal) -> evidence.proto (evidence) -> types.proto (duplicatevoteevidence & lightclientattackevidence)`